package.json

```diff
@@ -1,8 +1,9 @@
 {
-  "name": "Datepicker.js",
+  "name": "datepicker.js",
```

Comment:
I can't remember why, but at some point I changed the name to be capitalized. I actually now prefer it without the capital, but I'd like to be consistent. If you could make the similar changes to [this commit](https://github.com/wwilsman/Datepicker.js/commit/e25dddbcc2381041c16ac423b65f7698b56b6233), I'll change the repo name as well so that everything is `datepicker.js`.

Or we can change this back to `Datepicker.js` and I'll cut a release with the typescript entry to [this npm package](https://www.npmjs.com/package/Datepicker.js)

Reply:
I think capitalized name is a good idea because the name is also a file path, and you know, if I use it on windows, they will be same.
I will guess you change the name to be capitalized because it same as the name this lib exported.
I will change the rest name as you mentioned later.

Reply:
I checked [that commit](https://github.com/wwilsman/Datepicker.js/commit/e25dddbcc2381041c16ac423b65f7698b56b6233) and find out that may be there is no any other names should be changed  `Datepicker` to `datepicker.js`. Just the name in `package.json` should be changed as I commited.

Reply:
The library name may be `Datepicker.js`, but in `npm` it would be `datepicker.js`, it's OK, many other packages are doing the same thing, like famous [jQuery](https://www.npmjs.com/package/jquery)